Components/badges/styles.css

```diff
@@ -0,0 +1,285 @@
+* {
+    font-family: 'Lato', sans-serif;
+
+}
+
+
+body {
+    margin: 0;
+    padding: 0;
+
+
+}
+
+:root {
+    --primary-color: #63DCCD;
+    --secondary-color: #266350;
+}
```

Comment:
Don't declare variable colors in every other css file. Instead, create a separate file for all the variables. This will help in the maintainability